vision/google/cloud/vision/annotations.py

```diff
@@ -89,8 +89,11 @@ def from_api_repr(cls, response):
         annotations = {}
         for feature_type, annotation in response.items():
             curr_feature = annotations.setdefault(_KEY_MAP[feature_type], [])
-            curr_feature.extend(
-                _entity_from_response_type(feature_type, annotation))
+            entity = _entity_from_response_type(feature_type, annotation)
+            if isinstance(entity, list):
+                curr_feature.extend(entity)
+            else:
+                annotations[_KEY_MAP[feature_type]] = entity
```

Comment:
It didn't gain an entry, it's just handling for setting 'properties' to not be a list like the others so it matches the protobuf version.